src/js/post-911-gib-status/routes.jsx

```diff
@@ -1,8 +1,13 @@
-import Post911GIBStatusApp from './containers/Post911GIBStatusApp';
+import React from 'react';
+import { Route } from 'react-router';
 
-const routes = {
-  path: '/',
-  component: Post911GIBStatusApp
-};
+import PrintPage from './containers/PrintPage';
+
+const routes = [
```

Comment:
The only way I knew how to add a separate route for the print page was to change how we're handling our routes. In doing so, I tried to add a route for the `Main` component, where the path would just be `''`, but still having trouble getting that to work. So I'm still just rendering the `Main` component explicitly in the top level component. I think this would be a good topic for the next front end eng sync, since I'm pretty confused on how these routes work.